packages/inspector-extension/src/index.ts

```diff
@@ -270,7 +270,9 @@ const notebooks: JupyterFrontEndPlugin<void> = {
 
       // Listen for active cell changes.
       parent.content.activeCellChanged.connect((sender, cell) => {
-        handler.editor = cell && cell.editor;
+        cell?.ready.then(() => {
+          handler.editor = cell && cell.editor;
```

Comment:
I would add a check to avoid race condition in case the active cell changes quickly. And we can drop the test on `cell` as the `then` clause will not be triggered if `cell` is `null`.

```suggestion
          if (cell === parent.content.activeCell) {
            handler.editor = cell!.editor;
          }
```